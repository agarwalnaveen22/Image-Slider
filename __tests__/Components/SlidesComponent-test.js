import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Slides from '../../Components/SlidesComponent';

describe('test slides component', () => {
  const data = {
    item: {
      author: 'Danielle MacInnes',
    },
  };
  it('match snapshot', () => {
    const snap = renderer.create(<Slides item={data} />).toJSON();
    expect(snap).toMatchSnapshot();
  });
});
