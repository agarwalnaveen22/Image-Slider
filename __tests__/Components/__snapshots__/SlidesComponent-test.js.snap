// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test slides component match snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "justifyContent": "center",
      },
      undefined,
    ]
  }
>
  <View>
    <Text />
  </View>
  <Image
    source={
      Object {
        "uri": undefined,
      }
    }
  />
</View>
`;
